src/Resources/ToDo.svg

```diff
@@ -0,0 +1 @@
+<svg viewBox="0 0 96 96" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" id="Icons_Checklist_LTR" overflow="hidden"><g><path d="M23 14 73 14 73 82 23 82 23 14ZM17 88 79 88 79 8 17 8 17 88Z"/><rect x="50" y="23" width="17" height="4"/><rect x="50" y="39" width="17" height="4"/><rect x="50" y="71" width="17" height="4"/><rect x="50" y="55" width="17" height="4"/><path d="M43.8 20.8 41 18 34.4 24.6 31.8 22 29 24.8 34.4 30.2Z"/><path d="M43.8 36.8 41 34 34.4 40.6 31.8 38 29 40.8 34.4 46.2Z"/><path d="M43.8 52.8 41 50 34.4 56.6 31.8 54 29 56.8 34.4 62.2Z"/><path d="M43.8 68.6 41 65.8 34.4 72.4 31.8 69.8 29 72.6 34.4 78Z"/></g></svg>
```

Comment:
I'm getting an error when I try to deploy this: 

```Error processing 'edwon.okr-hub-dev' SVG file types in extension package are not supported. File at '/dist/fonts/ToDo.svg' is of type SVG, please convert it to a different format (like PNG or JPG) and try again.
```

Reply:
This is happening when you try to upload to a cloud instance?

Reply:
it happens when I deploy this branch to my personal dev/cloud instance

Reply:
and i get this error in my extension install log at `https://dev.azure.com/edwon/...`

Reply:
Okay, I'm looking into it